clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/ingestion/heartbeat/HeartbeatStat.java

```diff
@@ -9,8 +9,9 @@
 
 
 public class HeartbeatStat {
-  Map<String, WritePathLatencySensor> leaderSensors = new VeniceConcurrentHashMap<>();
-  Map<String, WritePathLatencySensor> followerSensors = new VeniceConcurrentHashMap<>();
+  Map<String, WritePathLatencySensor> readyToServeLeaderSensors = new VeniceConcurrentHashMap<>();
+  Map<String, WritePathLatencySensor> readyToServeFollowerSensors = new VeniceConcurrentHashMap<>();
+  Map<String, WritePathLatencySensor> catchingUpFollowerSensors = new VeniceConcurrentHashMap<>();
```

Comment:
I just want to confirm my understanding -> since current version offline->standby is blocking, there is no chance leader is in (after EOP, before COMPLETED) status right? So in theory we don't need to do the same for leader.

Reply:
More importantly, we should NEVER squelch lag for leader for a current version.  The reason why is because so long as the leader isn't on the tail of the RT, we're lagging.  So it's important to always report this for the sake of accurate SLA reporting.

Reply:
That's true. Thanks for the confirmation.